src/dguweb/web/controllers/search_controller.ex

```diff
@@ -6,7 +6,7 @@ defmodule DGUWeb.SearchController do
   end
 
   def search(conn, %{"q" => query}) do
-    q = query |> String.replace(" ","+")
+    q = query |> String.replace(" ","+") |> String.replace("-", "%2D")
     result = Repo.search(q)
```

Comment:
Yes, there is, and probably, but not too worried atm as I am going to switch out the library we use at some point to take care of all this.